packages/grafana-data/src/datetime/rangeutil.ts

```diff
@@ -134,6 +137,17 @@ export function describeTextRange(expr: string): TimeOption {
   return opt;
 }
 
+// TODO: Should we keep these format presets somewhere common?
```

Comment:
I think we should, yes. i'm ok with merging without making the decision of where. i'd *like* them to be in grafana-data, but how things are currently working they'd probably need to be in grafana-i18n.